plugins/inputs/statsd/statsd.go

```diff
@@ -322,6 +335,7 @@ func (s *Statsd) Start(ac telegraf.Accumulator) error {
 	s.counters = make(map[string]cachedcounter)
 	s.sets = make(map[string]cachedset)
 	s.timings = make(map[string]cachedtimings)
+	s.distributions = []cacheddistributions{}
```

Comment:
Same here to keep the same style as above
```suggestion
	s.distributions = make([]cacheddistributions, 0)
```